### PR TITLE
update for swift3.0

### PR DESCRIPTION
**Pull request by rigelxu**
I'm using your ‘DrawerController’，please update to swift3.0,thanks.

### CODE REVIEW

File: DrawerController/DrawerController.swift [hunk 1]
```diff
@@ -1511,42 +1511,37 @@ public class DrawerController: UIViewController, UIGestureRecognizerDelegate {
...
-        return CGRectContainsPoint(centerViewContentRect, point) && self.isPointContainedWithinNavigationRect(point) == false
+        return centerViewContentRect.contains(point) && self.isPointContainedWithinNavigationRect(point) == false
     }
     
-    func isPointContainedWithinLeftBezelRect(point: CGPoint) -> Bool {
-        var leftBezelRect = CGRectNull
-        var tempRect = CGRectNull
+    func isPointContainedWithinLeftBezelRect(_ point: CGPoint) -> Bool {
+        let leftBezelRect = CGRect.null
+        _ = childControllerContainerView.bounds.divided(atDistance: bezelRange, from: .minXEdge)
```
Comment (UweMeier) [inline DrawerController/DrawerController.swift]: shouldn't it rather be:
`var leftBezelRect = CGRect.null`
`(leftBezelRect, _) = self.childControllerContainerView.bounds.divided(atDistance: bezelRange, from: .minXEdge)`

File: DrawerController/DrawerController.swift [hunk 2]
```diff
@@ -1511,42 +1511,37 @@ public class DrawerController: UIViewController, UIGestureRecognizerDelegate {
...
     
-    func isPointContainedWithinRightBezelRect(point: CGPoint) -> Bool {
-        var rightBezelRect = CGRectNull
-        var tempRect = CGRectNull
-        
-        CGRectDivide(self.childControllerContainerView.bounds, &rightBezelRect, &tempRect, bezelRange, .MaxXEdge)
-        
-        return CGRectContainsPoint(rightBezelRect, point) && self.isPointContainedWithinCenterViewContentRect(point)
+    func isPointContainedWithinRightBezelRect(_ point: CGPoint) -> Bool {
+        let rightBezelRect = CGRect.null
```
Comment (UweMeier) [inline DrawerController/DrawerController.swift]: shouldn't it rather be:
`var rightBezelRect = CGRect.null`
`(rightBezelRect, _) = self.childControllerContainerView.bounds.divided(atDistance: bezelRange, from: .maxXEdge)`

**Comment by sascha** [comment]
I've released DrawerController 2.0.0 with Swift 2.3 support and DrawerController 3.0.0 with Swift 3.0 support.

**Comment by rigelxu** [comment]
Thanks

**Pull request closed without merging**